tests/unit/test_config.py

```diff
@@ -78,10 +78,15 @@ def mocked_env(monkeypatch, vcap_application, vcap_services):
     monkeypatch.setenv("VCAP_APPLICATION", vcap_application)
     monkeypatch.setenv("VCAP_SERVICES", vcap_services)
     monkeypatch.setenv("DEFAULT_CLOUDFRONT_ORIGIN", "None")
+    # TODO: validate listener arns are unique
     monkeypatch.setenv(
         "ALB_LISTENER_ARNS",
         "arn:aws:elasticloadbalancing:us-east-2:123456789012:listener/app/my-load-balancer/1234567890123456,arn:aws:elasticloadbalancing:us-east-2:123456789012:listener/app/my-load-balancer/1234567890123456",
     )
+    monkeypatch.setenv(
+        "DEDICATED_ALB_LISTENER_ARNS",
+        "arn:aws:elasticloadbalancing:us-east-2:123456789012:listener/app/my-other-load-balancer/1234567890123456,arn:aws:elasticloadbalancing:us-east-2:123456789012:listener/app/my-other-load-balancer/1234567890123456",
```

Comment:
Are we intentionally setting the value to the same exact listener ARN twice so that we can verify that de-duping works properly and only unique values are loaded into the config?

Reply:
this config is only used for testing that the config gets loaded, so it doesn't validate any de-dupe and isn't used outside this file.
the dedicated_alb_listener_arns is duplicated because I copied the alb_listener_arns and tweaked it slightly. I assume the alb_listener_arns is duplicated because I was just as lazy when I wrote that part

Reply:
☝🏻 I was wrong about deduplication - we are deduplicating and we are testing it here. 
I added comments to the fixture and tests about that.